internal/controller/vrg_volrep.go

```diff
@@ -181,6 +209,8 @@ func (v *VRGInstance) reconcileVolRepsAsSecondary() bool {
 		v.pvcStatusDeleteIfPresent(pvc.Namespace, pvc.Name, log)
 	}
 
+	v.reconcileVolGroupRepsAsSecondary(&requeue, groupPVCs)
```

Comment:
Same comment as Primary, if a PVC has undergone any skip/requeue then its VGR processing may need to happen later.